lib/NativeCode/DynamicLibraryLoaderHelper/ManagedPluginCode/Application.cs

```diff
@@ -0,0 +1,44 @@
+﻿/*
+ * Copyright (c) 2024 PlayEveryWare
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+
+namespace PlayEveryWare.EpicOnlineServices
+{
+    /// <summary>
+    /// This class is defined here as a stand-in for UnityEngine.Application,
+    /// so that files compiled outside of the Unity Editor that reference that
+    /// class can still be compiled properly.
+    /// </summary>
+    internal static class Application
+    {
+        /// <summary>
+        /// This is the path of the streaming assets directory relative to the
+        /// output directory of this class library.
+        /// </summary>
+        public static readonly string streamingAssetsPath = @"..\..\..\..\Assets\StreamingAssets\";
+
+        /// <summary>
+        /// For the purposes of this class library, this field member must be
+        /// present - but it's value is not utilized, so it is being left empty.
+        /// </summary>
+        public static readonly string temporaryCachePath = string.Empty;
```

Comment:
The reason I am not concerned about this is twofold:

1. This particular field member only needs to exist so that the code can compile
2. The _meaning_ of this value is irrelevant, since this is one of the _only two_ code files that this project owns - and this project will only be used to test the verisimilitude of the config values parsed by managed code and unmanaged code (as in - this file doesn't exist outside of a unit test).